arch/arm64/boot/dts/freescale/Makefile

```diff
@@ -52,6 +52,10 @@ dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1028a-qds-9999.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8dxl-evk.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mm-beacon-kit.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mm-data-modul-edm-sbc.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mm-crocodile-no-m4.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mm-crocodile-rev-b.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mm-crocodile-rev-b-rwk.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mm-crocodile-rev-c.dtb
```

Comment:
Where are the old dts file names?

Reply:
good question - was asking myself that... shouldn't they have been there already?
(only found them in the fitimage recipe...)